S12-methods/multi.t

```diff
@@ -115,7 +113,7 @@ is Bar.new.a("not an Int"), 'Any-method in Foo';
     is $m.d( '7' ), 'string',  'dispatch to other role';
     is $m.d( 1.2 ), 'any',     'dispatch to the class with the roles';
 
-    my @multi_method = $m.^methods.grep({ ~$_ eq 'd' });
+    my @multi_method = $m.^methods.grep({ .gist eq 'd' });
```

Comment:
Why not `.name` instead of `.gist` ?

Reply:
I was to tired to check it deeper, but basically you back up my yesterday thoughts. Will commit the change later today.